workshop.pl

```diff
@@ -665,6 +669,12 @@ sub arg_handler {
                 }
             }
         }
+
+        if ($found_req == 0) {
+            logger('info', "failed\n", 2);
```

Comment:
why do we have an info logger type here ?

Reply:
There are 'info', 'verbose', and 'debug' log-levels.  The normal logging is done as 'info' level and the "failed" message is the finishing state of a previous 'info' level message.  The 'error' level is reserved for errors and is emitted regardless of what the current log-level is.